[2023-04-26T09:12:13.345+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-04-26T09:12:10.960469+00:00 [queued]>
[2023-04-26T09:12:13.352+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available manual__2023-04-26T09:12:10.960469+00:00 [queued]>
[2023-04-26T09:12:13.353+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T09:12:13.353+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-04-26T09:12:13.353+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T09:12:13.361+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2023-04-26 09:12:10.960469+00:00
[2023-04-26T09:12:13.366+0000] {standard_task_runner.py:55} INFO - Started process 2217 to run task
[2023-04-26T09:12:13.370+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'is_api_available', 'manual__2023-04-26T09:12:10.960469+00:00', '--job-id', '234', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpv7bd20gb']
[2023-04-26T09:12:13.372+0000] {standard_task_runner.py:83} INFO - Job 234: Subtask is_api_available
[2023-04-26T09:12:13.423+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.is_api_available manual__2023-04-26T09:12:10.960469+00:00 [running]> on host c452347a64f2
[2023-04-26T09:12:13.469+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2023-04-26T09:12:10.960469+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-26T09:12:10.960469+00:00
[2023-04-26T09:12:13.470+0000] {http.py:120} INFO - Poking: api/
[2023-04-26T09:12:13.478+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2023-04-26T09:12:13.480+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2023-04-26T09:12:14.375+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2023-04-26T09:12:14.399+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=is_api_available, execution_date=20230426T091210, start_date=20230426T091213, end_date=20230426T091214
[2023-04-26T09:12:14.460+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-04-26T09:12:14.486+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
